dissect/target/plugins/browsers/chrome.py

```diff
@@ -17,25 +18,33 @@ class ChromePlugin(ChromiumMixin, Plugin):
         # Windows
         "AppData/Local/Google/Chrome/User Data/Default",
         "AppData/Local/Google/Chrome/continuousUpdates/User Data/Default",
-        "Local Settings/Application Data/Google/Chrome/User Data/Default",
```

Comment:
Why was this removed?

Reply:
But now it won't work on systems that don't have `AppData` but do have `Local Settings`. I know that's incredibly unlikely (XP running Chrome probably?), but IMO a better solution would be to perhaps use `.resolve()` to get a full absolute path, and filter out duplicate paths in the iteration process.